provision/sdk/sdk.go

```diff
@@ -0,0 +1,318 @@
+package sdk
```

Comment:
This should be added to Mainflux SDK, right? I do not see the point to have a separate SDK per service, we Mainflux SDK should be used for the whole system. We recently added Boostrap commands to it, this also should be transferred over there.

Reply:
this package is provision specific wrapper around mainflux sdk

Reply:
Can it be useful for other applications, and if yes why not putting it available to public though one unified SDK library?